starter-code/src/clue.js

```diff
@@ -1,94 +1,164 @@
+  occupation: "Socialité"
+}
+
+const mrMustard = {
+  first_name: "Jack",
+  last_name: "Mustard",
+  color: "yellow",
+  description: "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+}
 
 // Weapons
+const rope= {
+  name: "rope",
+  weight: 10,
+}  
+
+const knife= {
+  name: "knife",
+  weight: 8,
+}  
+
+const candlestick= {
+  name: "candlestick",
+  weight: 2,
+}  
+
+const dumbbell= {
+  name: "dumbbell",
+  weight: 30,
+}  
+
+const poison= {
+  name: "poison",
+  weight: 2,
+}  
+
+const axe= {
+  name: "axe",
+  weight: 15,
+}  
+
+const bat= {
+  name: "bat",
+  weight: 13,
+}  
+
+const trophy= {
+  name: "trophy",
+  weight: 25,
+}  
+
+const pistol= {
+  name: "pistol",
+  weight: 20,
+}  
+
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
 
 // Rooms
-name: Dining Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const diningRoom = { name: "Dining Room" }
+const conservatory = { name: "Conservatory" } 
+const kitchen = { name: "Kitchen" }
+const study = { name: "Study" }
+const library = { name: "Library" }
+const billiardRoom = { name: "Billiard Room" }
+const lounge = { name: "Lounge" }
+const ballroom = { name: "Ballroom" }
+const hall = { name: "Hall" }
+const spa = { name: "Spa" }
+const livingRoom = { name: "Living Room" }
+const observatory = { name: "Observatory" }
+const theater = { name: "Theater" }
+const guestHouse = { name: "Guest House" }
+const patio = { name: "Patio" }
 
 // Characters Collection
 const charactersArray = [];
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard);
 
 // Rooms Collection
 const roomsArray = [];
+roomsArray.push(diningRoom, conservatory, kitchen, study, library, billiardRoom, lounge, ballroom, hall, spa,livingRoom, observatory, theater, guestHouse, patio);
 
 // Weapons Collection
 const weaponsArray = [];
+weaponsArray.push(rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol);
+
+//return randomly one of the elements of the array
+function randomSelector (array) {
+  if (array.length === 0) {
+    return undefined
+  }
+  return array[Math.floor(Math.random() * array.length)]
+}
+
+function pickMystery() {
+  const suspect = randomSelector(charactersArray)
+  const weapon = randomSelector(weaponsArray)
+  const room = randomSelector(roomsArray)
+
+  const cards = {
+    suspect,
+    weapon,
+    room
+  }
+  return cards
```

Comment:
En vez de crear una variable para después devolverla, es mejor si directamente devolvéis el objeto con lo que queréis: 
```
return {
    suspect,
    weapon,
    room
}
```